infra/src/main/resources/exception/exceptionInfo.json

```diff
@@ -178,5 +178,11 @@
     "id": 30,
     "reason": "The specified ErrorHandler Action is unsupported.",
     "type": "USER"
+  },
+  {
+    "exception": "com.redhat.service.smartevents.infra.exceptions.definitions.platform.UnclassifiedException",
+    "id": 31,
+    "reason": "An Exception has not been mapped to an ExternalUserException. The raw error is shown.",
+    "type": "PLATFORM"
   }
```

Comment:
Reading this `reason`, what is the difference between `UnclassifiedException` and `UnclassifiedConstraintViolationException`? Do we need both?

Reply:
Well.... @jomarko one represents an internal failure if a Java Bean Validator returns a `ConstrainViolation` that has not been mapped to an `ExternalUserException`. The other is for when we don't map a `ExternalUserException` to a `BridgeError` (that is returned as a `ErrorResponse`).... The two IMO allows developers to easier disambiguate _why_ either of the errors may have been thrown; but they do serve similar purposes. i.e. to return a `ErrorResponse` structure when an "internal" error cannot be mapped to anything else.